test/integration/react-streaming-and-server-components/test/index.test.js

```diff
@@ -230,15 +231,17 @@ describe('Functions manifest', () => {
       'server',
       'functions-manifest.json'
     )
-    const content = JSON.parse(await fs.readFile(functionsManifestPath, 'utf8'))
+    const content = JSON.parse(fs.readFileSync(functionsManifestPath, 'utf8'))
     const { pages } = content
     const pageNames = Object.keys(pages)
 
     const paths = ['/', '/next-api/link', '/routes/[dynamic]']
     paths.forEach((path) => {
+      const { runtime, files } = pages[path]
       expect(pageNames).toContain(path)
-      expect(pages[path].runtime).toBe('web')
-      expect(pages[path].files.every((f) => f.startsWith('server/'))).toBe(true)
+      // Runtime of page `/` is undefined since it's configured as nodejs.
+      expect(runtime).toBe(path === '/' ? undefined : 'web')
```

Comment:
Should we write the `nodejs` value to the manifest to be explicit and ensure the runtime field is always set?

Reply:
filesystem api currently use nodejs version, if leave empty it will use the project confiugured node version. I guess we can't leave it as `"nodejs"`

Reply:
Definitely agree on having it always filled up